src/Nokia_LCD.cpp

```diff
@@ -112,6 +112,19 @@ void Nokia_LCD::begin() {
     sendCommand(0x0C);  // Set display control, normal mode.
 }
 
+void Nokia_LCD::couple() {
+    if (!coupled) {
+        digitalWrite(kCe_pin, LOW);
+        coupled = true;
+    }
```

Comment:
Any particular reason why the `if` statement is _not_ skipped?

If the claim is performance, then it should be backed up by measurements to determine whether the increase in complexity justifies it.

```suggestion
    digitalWrite(kCe_pin, LOW);
    mCoupled = true;
```